migrations/20180215163030_add_mortgage_price.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE MortgageDetails ADD COLUMN mortgagePrice int(11);
```

Comment:
Where is the corresponding .down file?

Reply:
And since you're editing might as well add a new line at the end of the up file :stuck_out_tongue:

Reply:
Done